tex/latex/biblatex/lbx/french.lbx

```diff
@@ -439,25 +439,25 @@
   thiscite         = {{sp\'ecialement}{sp\'ec\adddot}},
   seenote          = {{voir note}{cf\adddotspace note}},
   quotedin         = {{cit\'e dans}{cit\'e dans}},
-  idem             = {{\mkbibemph{idem}}{\mkbibemph{idem}}},
-  idemsm           = {{\mkbibemph{idem}}{\mkbibemph{idem}}},
-  idemsf           = {{\mkbibemph{eadem}}{\mkbibemph{eadem}}},
-  idemsn           = {{\mkbibemph{idem}}{\mkbibemph{idem}}},
-  idempm           = {{\mkbibemph{eidem}}{\mkbibemph{eidem}}},
-  idempf           = {{\mkbibemph{eaedem}}{\mkbibemph{eaedem}}},
-  idempn           = {{\mkbibemph{eadem}}{\mkbibemph{eadem}}},
-  idempp           = {{\mkbibemph{eidem}}{\mkbibemph{eidem}}},
-  ibidem           = {{\mkbibemph{ibidem}}{\mkbibemph{ibid\adddot}}},
-  opcit            = {{\mkbibemph{op\adddotspace cit\adddot}}{\mkbibemph{op\adddotspace cit\adddot}}},
-  loccit           = {{\mkbibemph{loc\adddotspace cit\adddot}}{\mkbibemph{loc\adddotspace cit\adddot}}},
-  confer           = {{\mkbibemph{cf\adddot}}{\mkbibemph{cf\adddot}}},
-  sequens          = {{\mkbibemph{sq\adddot}}{\mkbibemph{sq\adddot}}},
-  sequentes        = {{\mkbibemph{sqq\adddot}}{\mkbibemph{sqq\adddot}}},
-  passim           = {{\mkbibemph{passim}}{\mkbibemph{pass\adddot}}},
-  see              = {{\mkbibemph{cf\adddot}}{\mkbibemph{cf\adddot}}},
-  seealso          = {{\mkbibemph{cf\adddotspace}aussi}{\mkbibemph{cf\adddotspace}aussi}},
-  backrefpage      = {{\mkbibemph{cf\adddotspace}page}{\mkbibemph{cf\adddotspace}p\adddot}},
-  backrefpages     = {{\mkbibemph{cf\adddotspace}pages}{\mkbibemph{cf\adddotspace}p\adddot}},
+  idem             = {{idem}{idem}},
+  idemsm           = {{idem}{idem}},
+  idemsf           = {{eadem}{eadem}},
+  idemsn           = {{idem}{idem}},
+  idempm           = {{eidem}{eidem}},
+  idempf           = {{eaedem}{eaedem}},
+  idempn           = {{eadem}{eadem}},
+  idempp           = {{eidem}{eidem}},
+  ibidem           = {{ibidem}{ibid\adddot}},
+  opcit            = {{op\adddotspace cit\adddot}{op\adddotspace cit\adddot}},
+  loccit           = {{loc\adddotspace cit\adddot}{loc\adddotspace cit\adddot}},
+  confer           = {{cf\adddot}{cf\adddot}},
+  sequens          = {{sq\adddot}{sq\adddot}},
+  sequentes        = {{sqq\adddot}{sqq\adddot}},
+  passim           = {{passim}{{pass\adddot}},
```

Comment:
I guess this should be
```suggestion
  passim           = {{passim}{pass\adddot}},
```

Reply:
Oops, little mistake while removing \mkbibemph… I correct it.